### ISSUE TITLE
OSM Biome & Vegetation bug

### ISSUE THREAD

**Issue by sys0pt** [orangeadam3/terra121#34]
![2020-03-27_00 26 33](https://user-images.githubusercontent.com/62664272/77706118-c56f5300-6fc1-11ea-9a74-fa942f985084.png)

Greece is mostly Mesa and looks like canyon. This specific location should be actually green and with trees, also stone cliffs.

**Comment by orangeadam3** [comment]
Yeah the biomes espically are an approxomation, the biome classifier needs to be refined for the next version. It looks like the trees are spawning properly from your image, however.

**Comment by Skyguy92** [comment]
I also am having a weird issue except the trees are not working right. For my area, the trees should be more spruce like than the swamp. is this something with the algorithm? 

https://gyazo.com/99c0e55e4f8350d83a570a170f09fdec
![2020-03-26_20 31 12](https://user-images.githubusercontent.com/29076439/77712081-c04ddc00-6fa0-11ea-9cee-332c59fbc838.png)

**Comment by orangeadam3** [comment]
The tree type is based on biome so it goes back to the poor biome classification. It would be nice to find a dataset for soil and climate of biomes.

**Comment by ELIAREy** [comment]
There are other issues with biomes that I have noticed. Basically, the entire Peruvian coastline is covered in Swampland in this mod ; it's supposed to be the opposite. Peru on its coast (especially Lima) has a very dry climate, but very often covered in a humid fog, and it never rains. So I would change the soil to sandstone and stone and the climate would be the same as a desert.
Also, sand deserts that are suposedly sand in Namibia and Egypt were Mesa biomes. Same thing with the largest sand desert  in the American continent (the famous Cerro Blanco) which is in Ica, Peru.
Finally, forests which are at high altitudes (1500+ meters) in the Alps are supposed to be Taiga-like, with spruce trees, but instead, it's covered with "normal" oak and birch trees.
Also I forgot : snow-covered mountains like the Huascaran (culminating point of Peru) and Mont-Blanc are not at all covered in snow : I think it would be a good idea to try and coordinate satellite images with snow-covered regions so at least we have snowy biomes at these altitudes if you considerer we are constantly in the summer.
This will be a lot of work I imagine, and these are only a few specific examples so I don't know how to make it more world-wide, but I wish you good luck !
